web/src/app/admin/connectors/alation/page.tsx

```diff
@@ -0,0 +1,311 @@
+    error: isCredentialsError,
+  } = useSWR<Credential<AlationCredentialJson>[]>(
+    "/api/manage/credential",
+    fetcher,
+  );
+
+  if (
+    (!connectorIndexingStatuses && isConnectorIndexingStatusesLoading) ||
+    (!credentialsData && isCredentialsLoading)
+  ) {
+    return <LoadingAnimation text="Loading" />;
+  }
+
+  if (isConnectorIndexingStatusesError || !connectorIndexingStatuses) {
+    return <div>Failed to load connectors</div>;
+  }
+
+  if (isCredentialsError || !credentialsData) {
+    return <div>Failed to load credentials</div>;
+  }
+
+  const alationConnectorIndexingStatuses = connectorIndexingStatuses.filter(
+    (connectorIndexingStatus) =>
+      connectorIndexingStatus.connector.source === "alation",
+  );
+  const alationCredential = credentialsData.filter(
+    (credential) => credential.credential_json?.alation_refresh_token,
+  )[0];
+
+  return (
+    <>
+      {popup}
+      <h2 className="font-bold mb-2 mt-6 ml-auto mr-auto">
+        Indexing an Alation Server
+      </h2>
+      <p>This connector allows for indexing objects from an Alation server.</p>
+      <p className="mb-4">
+        Today we support indexing these specific object types:
+      </p>
+      <ul className="list-disc ml-8 mb-4">
+        <li>Articles</li>
+        <li>Tables</li>
+        <li>Schemas</li>
+        <li>Columns</li>
+      </ul>
+      <p>
+        By default the connector extracts everything. You can limit the number
+        of objects extracted by setting the "Maximum objects to index per Object
+        Type" value below.
+      </p>
+
+      <h2 className="font-bold mb-2 mt-6 ml-auto mr-auto">
+        Step 1: Provide your Alation Refresh Token
+      </h2>
+
+      {alationCredential ? (
+        <>
+          <div className="flex mb-1 text-sm">
+            <p className="my-auto">Existing Refresh Token: </p>
+            <p className="ml-1 italic my-auto max-w-md truncate">
+              {alationCredential.credential_json?.alation_refresh_token}
+            </p>
+            <button
+              className="ml-1 hover:bg-gray-700 rounded-full p-1"
+              onClick={async () => {
+                if (alationConnectorIndexingStatuses.length > 0) {
+                  setPopup({
+                    type: "error",
+                    message:
+                      "Must delete all connectors before deleting credentials",
+                  });
+                  return;
+                }
+                await deleteCredential(alationCredential.id);
+                mutate("/api/manage/credential");
+              }}
+            >
+              <TrashIcon />
+            </button>
+          </div>
+        </>
+      ) : (
+        <>
+          <p className="text-sm">
+            To use the Alation connector, first follow the guide{" "}
+            <a
+              className="text-blue-500"
+              href="https://docs.danswer.dev/connectors/alation#setting-up"
```

Comment:
Please add PR to docs repo, I'll wait to approve this one for when that PR exists as well

https://github.com/danswer-ai/danswer-docs